### PR TITLE
new starting point

### PR DESCRIPTION
**Pull request by Captainmolt**
hey y'all. Look at Weston's code. (I told him what to do)(give me credit)(I don't know how to code java and I'm too lazy to search it up leave me alone)

### CODE REVIEW

File: Interface.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class Interface {
 
     sendClickToServer(row, col) {
         // Fire and forget - wait on pulse to actually pick up the change.
-        fetch(SERVER_URL + CLICK_SUFFIX + "?x=" + col + "&y=" + row);
+        fetch(SERVER_URL + CLICK_SUFFIX + "?x=" + col + "&y=" + row);
```
Comment (Imperii) [inline Interface.js]: ws

**Comment by HippoShark** [comment]
lgtm

**Comment by HippoShark** [comment]
lgtm

**Comment by Imperii** [comment]
Lowe

**Pull request merged**